### PR TITLE
[SymbolGraph] Show get/set on property/subscript full declarations

### PR DESCRIPTION
**Pull request by bitjammer**
But don't show them in subheading contexts.

rdar://63233897

### CODE REVIEW

**Comment by bitjammer** [comment]
@swift-ci Please smoke test

**Comment by bitjammer** [comment]
@swift-ci Please smoke test

**Review by jackhl — CHANGES_REQUESTED** [review]
Can you add a test that a `let` stored property gets a `{get}` and a `var` stored property gets a `{get set}`?

File: lib/SymbolGraphGen/SymbolGraph.cpp [hunk 1]
```diff
@@ -76,6 +76,16 @@ SymbolGraph::getSubHeadingDeclarationFragmentsPrintOptions() const {
   auto Options = getDeclarationFragmentsPrintOptions();
   Options.ArgAndParamPrinting =
     PrintOptions::ArgAndParamPrintingMode::ArgumentOnly;
+
+  //--------------------------------------------------------------------------//
+  // Although we allow these in the full declaration presentation,
+  // we don't want to show these for even for protocol requirements
```
[the remark this answers is not in the document]
Comment (bitjammer) [reply lib/SymbolGraphGen/SymbolGraph.cpp:82]: That's some good copy pasta :).

[the remark this answers is not in the document]
Comment (bitjammer) [reply lib/SymbolGraphGen/SymbolGraph.cpp]: Fixed, thanks.

**Comment by bitjammer** [comment]
Stored properties don't show `{ get set }` -- see https://developer.apple.com/documentation/coregraphics/cgpoint/1456040-x.

**Comment by bitjammer** [comment]
@swift-ci Please smoke test.

**Comment by bitjammer** [comment]
Looks like an unrelated swift-driver failure. Will try again in a little while.

**Comment by bitjammer** [comment]
@swift-ci Please smoke test

**Comment by bitjammer** [comment]
Windows bot is out of space.

**Comment by bitjammer** [comment]
@swift-ci Please test Windows platform

**Comment by bitjammer** [comment]
@swift-ci Please test Windows platform

**Pull request merged**